includes/templates/reader-activation/login-form.php

```diff
@@ -16,7 +16,7 @@
 
 \do_action( 'woocommerce_before_customer_login_form' );
 ?>
-<div class="newspack-ui newspack-reader-auth__inline-wrapper">
+<div class="newspack-ui">
```

Comment:
```suggestion
<div class="newspack-ui newspack-reader-auth__inline-wrapper">
```

This CSS class should get re-added -- it's part of this fix and won't affect the block: https://github.com/Automattic/newspack-plugin/pull/3292